src/Driver/DriverFactory.php

```diff
@@ -14,12 +16,18 @@ class DriverFactory
      * It accepts a PECL connection, a PhpAmqpLib connection or an array following
      * the following schema :
      * [
-     *     'host' => '<hostValue>',
-     *     'port' => '<portValue>',
-     *     'user' => '<userValue>',
-     *     'pwd'  => '<pwdValue>'
+     *     'host'        => '<hostValue>',
+     *     'port'        => '<portValue>',
+     *     'user'        => '<userValue>',
+     *     'pwd'         => '<pwdValue>',
+     *     'ssl'         => true,                   // Optional: default to false
+     *     'ssl_options' => [                       // Optional: default to [], see https://www.php.net/manual/context.ssl.php
+     *         'capath'      => '/etc/ssl/certs',
+     *         'verify_peer' => true,
+     *     ],
```

Comment:
Suggestion vis-à-vis de [la remarque de @qpautrat](https://github.com/Evaneos/evaneos-ng/pull/11379/files#r408025009) :

```suggestion
     *     'ssl'         => [
     *         'enabled'     => false,
     *         'capath'      => '/etc/ssl/certs',
     *         'verify_peer' => true,
     *     ],
```

C'est ultra discutable ceci dit 🤷

Reply:
Je préfère laisser le flag en dehors des options qui sont passées directement au contexte SSL.